mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +258,20 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+class MCQBlockBlankThemeTest(MCQBlockTest):
+    """
+    Test MRQ/MCQ questions without the LMS theme which is on by default.
+    """
+    def __init__(self, *args, **kwargs):
+        super(MCQBlockBlankThemeTest, self).__init__(*args, **kwargs)
+
+    def setUp(self, *args, **kwargs):
+        super(MCQBlockBlankThemeTest, self).setUp(*args, **kwargs)
+        self._original_get_theme = MentoringBlock.get_theme
+        MentoringBlock.get_theme = lambda self: {'package': None, 'locations': []}
```

Comment:
Have you considered using the [`mock`](https://pypi.python.org/pypi/mock/) library to handle this? It's more standard, and it cleans itself up automatically after the test.
```python
def setUp(self):
    patcher = mock.patch.object(MentoringBlock, "get_theme",
                                return_value={'package': None, 'locations': []})
    patcher.start()
```
or, as a decorator:
```python
@mock.patch.object(MentoringBlock, "get_theme", return_value={'package': None, 'locations': []})
class MCQBlockBlankThemeTest(MCQBlockTest):
   pass
```